bot/exts/filtering/_filter_lists/domain.py

```diff
@@ -56,7 +58,7 @@ async def actions_for(
 
         triggers = await self[ListType.DENY].filter_list_result(new_ctx)
         ctx.notification_domain = new_ctx.notification_domain
-        unknown_urls = urls - {filter_.content for filter_ in triggers}
+        unknown_urls = urls - {filter_.content.lower() for filter_ in triggers}
```

Comment:
Without this, the phishing button would suggest adding `"example.com/abcdef"` to the comp-filter-list even if `"example.com/aBcDeF"` is already present in the filter list.

Reply:
Is there a reason we don't normalise urls to lowercase before adding them to the filter list?

Reply:
I'm not sure. One potential reason I can think of is that that normalizing URLs to lowercase might lead to loss of distinction between case-sensitive URLs (example.com/aBcD vs example.com/abcd) so we will not be able to visit the website ourselves later (, perhaps, for verification reasons). Although I guess  we could search for the message with the original link but that would be tedious.

Reply:
Fair, not something worth changing in this PR at least.